coding-concepts.js

```diff
@@ -37,23 +37,24 @@ const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:"Ruby"
+// b) Verify and explain: Output = "Ruby".  The reason for this is an array is different from a string and that the index values are not the individual characters but the string itself between the commas.  Since index = 1 when we console.log(languages[index]) we wanted to see what value was at index and since index = 1 it outputed "Ruby".
 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:["SATURDAY", "SUNDAY"]
+// b) Verify and explain: It came back with an error saying that "TypeError: weekendDays.toUpperCase is not a function".  After doing some research it is because the built-in method .toUpperCase has to be used inside a function and  console.log(weekendDays.toUpperCase()) is not a function so it is not possible to run this code.
```

Comment:
You can certainly use a function to iterate through each value in the array and perform an action (.toUpperCase()) on each string. The sole reason `.toUpperCase()` does not work here is that it is a built-in method for the string datatype. Right now weekendDays is an array because that is the value it is storing.